### ISSUE TITLE
It should be easier to see all submitted files in the inspect submission page

### ISSUE THREAD

**Issue by Mankro** [apluslms/a-plus#920]
The inspect submission page: teachers can access the inspect page from the exercise's submission list. (Inspect buttons on the right.)

Juha presents two alternative solutions below. Which one should we choose?

From @jsorva:

It’s good that the submission (student code file) is visible right away. 
However, only a single file is. When a submission consists of multiple 
files, which is not unusual, it’s often the case that I need to switch 
to a different file. This currently requires clicking on the dropdown 
menu, visually locating the file name, and clicking again.

Tabs with the file names as titles would simplify this. Alternatively, I 
would also be fine with seeing all the submitted files on the same page, 
much like what we now have in the model solution popups.

Internal ticket: https://rt.cs.aalto.fi/Ticket/Display.html?id=20169
